src/Symfony/Component/HttpKernel/Tests/EventListener/LocaleListenerTest.php

```diff
@@ -26,11 +25,11 @@
 
 class LocaleListenerTest extends TestCase
 {
-    private MockObject&RequestStack $requestStack;
+    private RequestStack $requestStack;
 
     protected function setUp(): void
     {
-        $this->requestStack = $this->createMock(RequestStack::class);
+        $this->requestStack = new RequestStack();
```

Comment:
Why this setup, I would use new RequestStack() directly for discoverability, no?